app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ActiveRecord::Base
+  validates :first_name, :last_name, presence: true
+  validates :email, format: { 
+      with: /\b[A-Z0-9._%a-z\-]+@(?:[A-Z0-9a-z\-]+\.)+[A-Za-z]{2,4}\z/ 
+      }, uniqueness: true
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.